modules/ROOT/attachments/admin-api.yaml

```diff
@@ -2249,6 +2284,48 @@ components:
           type: string
         port:
           type: integer
+    pre_restart_check_result:
+      type: object
+      description: Pre-restart check result.
+      properties:
+        risks:
+          $ref: '#/components/schemas/restart_risks'
+    restart_risks:
+      description: Partitions affected by the current broker restart, grouped by risk type. Each partition list is truncated according to the optional limit specified in the request.
+      properties:
+        rf1_offline:
+          type: array
+          items:
+            type: string
+            description: ntp
```

Comment:
@bashtanov we actually already have a components.schema definition for ntp: https://github.com/redpanda-data/docs/blob/api/modules/ROOT/attachments/admin-api.yaml#L4639 would it make sense to use a $ref here?

Reply:
The ntp definition linked describes an object with 3 separate key-value entries (key being `ns`, `topic`, `partition`), while here it is a combined string. So no, it would be incorrect to refer to this existing definition.